sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -17,19 +17,15 @@ public class DataFeedLiveTests : MetricsAdvisorLiveTestBase
         private const string DataSourceAccount = "account";
         private const string DataSourceAppId = "appId";
         private const string DataSourceClientId = "clientId";
-        private const string DataSourceClientSecret = "clientSecret";
```

Comment:
We don't really test secrets in live tests (we have mock tests for that). So removing these constants we don't need (we're just sending "secret" instead of these pre-defined strings).